src/components/UI/Modal/Modal/Modal.module.scss

```diff
@@ -7,7 +7,7 @@
   right: 0;
   bottom: 0;
   position: absolute;
-  z-index: 100000;
+  z-index: 5;
```

Comment:
Why specifically `5`?

Reply:
Because the `getStarknetWallet` modal (the extended one) is with `z-index:6`, we need it to be before our modal so users can click on it

Reply:
So why not 1?

Reply:
because I wanted it to be the high as possible, similar to what you wrote: 100000

Reply:
If no other view is "racing" on the z-index, let's keep it 1 for now